client/src/components/Page.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { t } from "@lingui/macro";
+import { withI18n } from "@lingui/react";
+import { i18n } from "@lingui/core";
+import helpers from "../util/helpers";
+
+const metadataWithoutI18n = {
+  keywords: i18n._(t`pay rent, rent, can't pay rent, may rent, may 1`),
+  description: i18n._(
+    t`Use this free tool from JustFix.nyc to research your building and investigate landlords! We use property ownership mapping to identify a landlord's portfolio and provide data to indicate potential tenant harassment and displacement.`
+  ),
+  siteName: i18n._(t`Who owns what in nyc?`),
+};
+
+const getMetadata = () => withI18n()(metadataWithoutI18n);
```

Comment:
Oh funky, I'm not sure if this function is actually doing anything useful, since `metadataWithoutI18n` is already a constant at this point, so its localized values are already in-place at the time the function is called?  Or am I misreading something...

Reply:
Ooops, yeah you're right. I think I swapped my logic here. The `metadataWithoutI18n` component should be made into a function, not this here. This goes along with my response above.